sql/table.go

```diff
@@ -88,12 +89,18 @@ func makeTableDesc(p *parser.CreateTable) (structured.TableDescriptor, error) {
 			}
 			if d.PrimaryKey {
 				index.Name = "primary"
+				desc.Indexes = append([]structured.IndexDescriptor{index}, desc.Indexes...)
+			} else {
+				desc.Indexes = append(desc.Indexes, index)
```

Comment:
Added a new boolean to specify that an index is primary. I've left the primary index prepend as is.